website/blog/2023-11-11-AgentEval/index.mdx

```diff
@@ -0,0 +1,151 @@
+
+| Criteria      | Description | Accepted Values|
+|-----------|-----|----------------|
+| Problem Interpretation      | Ability to correctly interpret the problem  | ["completely off", "slightly relevant", "relevant", "mostly accurate", "completely accurate"]|
+| Mathematical Methodology      | Adequacy of the chosen mathematical or algorithmic methodology for the question  | ["inappropriate", "barely adequate", "adequate", "mostly effective", "completely effective"] |
+| Calculation Correctness       | Accuracy of calculations made and solutions given  |   ["completely incorrect", "mostly incorrect", "neither", "mostly correct", "completely correct"]     |
+| Explanation Clarity       | Clarity and comprehensibility of explanations, including language use and structure  | ["not at all clear", "slightly clear", "moderately clear", "very clear", "completely clear"] |
+| Code Efficiency       |  Quality of code in terms of efficiency and elegance |["not at all efficient", "slightly efficient", "moderately efficient", "very efficient", "extremely efficient"] |
+| Code Correctness      | Correctness of the provided code  |  ["completely incorrect", "mostly incorrect", "partly correct", "mostly correct", "completely correct"]
+
+
+Then, after running QuantifierAgent, we obtained the results presented in Fig. 3, where you can see three models:
+* AgentChat
+* ReAct
+* GPT-4 Vanilla Solver
+
+Lighter colors represent estimates for failed cases, and brighter colors show how discovered criteria were quantified.
+
+![Fig.3: Results based on overall mathp roblems dataset `_s` stands for successful cases, `_f` - stands for failed cases](img/math-problems-plot.png)
+<p align="center"><em>Fig.3 presents results based on overall mathproblems datase `_s` stands for successful cases, `_f` - stands for failed cases</em></p>
+
+We note that while applying agentEval to math problems, the agent was not exposed to any ground truth information about the problem. As such, this figure illustrates an estimated performance of the three different agents, namely, Autogen (blue), Gpt-4 (red), and ReAct (green). We observe that by comparing the performance of any of the three agents in successful cases (dark bars of any color) versus unsuccessful cases (lighter version of the same bar), we note that AgentEval was able to more accurately estimate the performance of successful cases than that of failed ones. This observation verifies AgentEval's ability for task utility prediction. Additionally, AgentEval allows us to go beyond just a binary definition of success, enabling a more in-depth comparison between successful and failed cases.
+
+It's important not only to identify what is not working but also to recognize what and why actually went well.For example, Explanation Clarity turned out to be correlated with the length of the solution and all solutions master _Task Understanding_ criteria.

```

Comment:
```suggestion
It's important not only to identify what is not working but also to recognize what and why actually went well. For example, Explanation Clarity turned out to be correlated with the length of the solution and all solutions master _Task Understanding_ criteria.
```
"Explanation Clarity turned out to be correlated with the length of the solution " Is it done in a different analysis and not shown in the plot? It'll be good to clarify.

Reply:
we have not included them in this blog post yet. Therefore, I removed this sentence to avoid any confusion